package.json

```diff
@@ -55,7 +57,7 @@
     "validate-commit-msg": "^2.14.0"
   },
   "peerDependencies": {
-    "babel-core": "6.*"
+    "babel-core": "6.* || ^7.0.0-bridge.0"
```

Comment:
If you want to make a breaking change, I can update to `"@babel/core": "7.*"` ?

Reply:
Yeah, let's do that. I'll publish this as a major version bump anyways.

Reply:
done :)